src/index.tsx

```diff
@@ -54,10 +55,18 @@ ReactDOM.render(
         {routes.map((route) => {
           const Component = route.component;
 
+          // Используемый компонент для роута
+          let UsedRoute: typeof Route | typeof ProtectedRoute = Route;
```

Comment:
🧐
Не очень нравится `Used` в названии, мб вот так сделаем?
```suggestion
          let RouteComponent: typeof Route | typeof ProtectedRoute = Route;
```